src/Bridge/Symfony/Routing/ApiLoader.php

```diff
@@ -221,6 +224,10 @@ private function addRoute(RouteCollection $routeCollection, string $resourceClas
             }
         }
 
+        if (!isset($operation['identifiedBy'])) {
+            $operation['identifiedBy'] = null !== $this->identifiersExtractor ? $this->identifiersExtractor->getIdentifiersFromResourceClass($resourceClass) : ['id'];
+        }
```

Comment:
this is wrong IMO and should be computed by the ResourceMetadataFactory, do you have an idea on how to use `PropertyMetadataFactory` inside a `ResourceMetadataFactory` @dunglas  ?